test/cs_guide_web/views/brand_view_test.exs

```diff
@@ -21,6 +21,18 @@ defmodule CsGuideWeb.BrandViewTest do
     ]
   }
 
+  @venues [
+    %CsGuide.Resources.Venue{
+      venue_types: [%{name: "retailer"}]
+    },
+    %CsGuide.Resources.Venue{
+      venue_types: [%{name: "wholesaler"}]
+    },
+    %CsGuide.Resources.Venue{
+      venue_types: [%{name: "type1"}]
```

Comment:
would prefer if we refactor `type1` into something more human-friendly ...
https://github.com/club-soda/club-soda-guide/search?q=type1&unscoped_q=type1

(_"works" for now, but would be good to make it more beginner-friendly..._)